base/deprecated.jl

```diff
@@ -1632,6 +1632,12 @@ function SymTridiagonal(dv::AbstractVector{T}, ev::AbstractVector{S}) where {T,S
     SymTridiagonal(convert(Vector{R}, dv), convert(Vector{R}, ev))
 end
 
+# issue #22791
+@deprecate_binding select partialsort
```

Comment:
OK, I don't know the reason for this difference but that's enough to simply use `@deprecate`. PR updated.